release_build_files/readme.md

```diff
@@ -587,6 +587,9 @@ code.
 	with version 8.x of the Firebase iOS SDK.
     -   General (iOS): A Database URL is no longer required to be present in
         GoogleService-Info.plist when not using the Real Time Database.
+    -   Firestore: Added `Firestore::LoadBundle` to enable loading Firestore Data
+        Bundles to the SDK cache. `Firestore::NamedQuery` will be availabe in next
```

Comment:
Hmm, I think it should be `s/to/into/` (i.e., `loading... into the cache`). @jonsimantov Jon, which one looks better to you?

Reply:
`into` is better, IMO.